packages/core/src/domain/session/sessionStoreOperations.ts

```diff
@@ -118,3 +120,12 @@ function next(sessionStore: SessionStoreStrategy) {
     processSessionStoreOperations(nextOperations, sessionStore)
   }
 }
+
+export function createLock(): string {
+  return generateUUID() + LOCK_SEPARATOR + timeStampNow()
```

Comment:
Why do we use a UUID for the lock? Does the UUID any meaning?

Reply:
If the lock was a constant value (ex: `lock=true`), when two instances of the SDK tried to lock at the same time, both would consider the lock to be successful and proceed to modify the cookie.

With a random lock id, each SDK instance write the lock then read the cookie, and consider it to be a success only if it's still the same value.